src/loaders/ObjectLoader.js

```diff
@@ -564,13 +564,19 @@ Object.assign( ObjectLoader.prototype, {
 				}
 
 				if ( data.format !== undefined ) texture.format = data.format;
+				if ( data.type !== undefined ) texture.type = data.type;
+				if ( data.encoding ) texture.encoding = data.encoding;
 
 				if ( data.minFilter !== undefined ) texture.minFilter = parseConstant( data.minFilter, TEXTURE_FILTER );
 				if ( data.magFilter !== undefined ) texture.magFilter = parseConstant( data.magFilter, TEXTURE_FILTER );
 				if ( data.anisotropy !== undefined ) texture.anisotropy = data.anisotropy;
+				if ( data.mipmaps !== undefined ) texture.mipmaps = data.mipmaps;
```

Comment:
AFAIK, these are manually created mipmaps. How can these data be represented in serialized JSON?

Reply:
Yup, it's for manual mipmaps generation. According to the source code, .mipmaps format is either one of 

    [ image1, image2, ... ]

    [ { width: width1, height: height1, data: typedArray1 },
      { width: width2, height: height2, data: typedArray2 } ]

Then, it should be like this in JSON?

    [ uuid1, uuid2 ]
    images = [
        { uuid: uuid1, ... },
        { uuid: uuid2, ... },
    ]

    [
      { width: width1, height: height1, data:  [0, 1, 2, ... ], type: typedArrayType1 },
      { width: width2, height: height2, data:  [0, 1, 2, ... ], type: typedArrayType2 },
    ]

Reply:
More precisely, for this style

    [ image1, image2, ... ]

image* should be ImageData / HTMLImageElement / HTMLCanvasElement / HTMLVideoElement / ImageBitmap.

For this style

    [ { width: width1, height: height1, data: typedArray1 },
      { width: width2, height: height2, data: typedArray2 } ]

typedArray* should be ArrayBufferView

https://developer.mozilla.org/en-US/docs/Web/API/WebGLRenderingContext/texImage2D
https://github.com/mrdoob/three.js/blob/dev/src/renderers/webgl/WebGLTextures.js

Any type of ImageData / HTMLImageElement / HTMLCanvasElement / HTMLVideoElement / ImageBitmap can be serialized by `getDataURL()` of `Texture.toJSON()`.

https://github.com/mrdoob/three.js/blob/dev/src/textures/Texture.js#L133

Reply:
If @mrdoob thinks it worth, then let's give it a try.

Reply:
Implemented `.mipmaps` serialization so far.